test/scale.time.tests.js

```diff
@@ -245,111 +220,131 @@ describe('Time scale tests', function() {
 	});
 
 	it('should build ticks using the config unit', function() {
-		var scaleID = 'myScale';
-
 		var mockData = {
 			labels: ['2015-01-01T20:00:00', '2015-01-02T21:00:00'], // days
 		};
 
-		var mockContext = window.createMockContext();
 		var config = Chart.helpers.clone(Chart.scaleService.getScaleDefaults('time'));
 		config.time.unit = 'hour';
-		var Constructor = Chart.scaleService.getScaleConstructor('time');
-		var scale = new Constructor({
-			ctx: mockContext,
-			options: config, // use default config for scale
-			chart: {
-				data: mockData
-			},
-			id: scaleID
-		});
 
-		// scale.buildTicks();
-		scale.update(400, 50);
+		var scale = createScale(mockData, config);
+		scale.update(5000, 200);
```

Comment:
It just needed to be wide enough to not be in danger of having ticks dropped for overlapping. Looking again this can be revised down to about `2500` before the last two ticks might overlap. 

Alternatively we can have a narrower graph and update the `expect` to remove the penultimate tick.